docs/cluster-configuration.md

```diff
@@ -944,9 +944,9 @@ Please correctly consider the implications of this option before setting it.
 `BlockUninstallIfWorkloadsExist` is the default behaviour.
 
 
-## Cluster level EvictionStrategy
+## Cluster level evictionStrategy
```

Comment:
I think that at lease the title be a proper English title, not the actual API field (which is specified in the paragraph). I would suggest "_eviction strategy_" to align with other titles in the docs.